test/unit/server-statefulset.bats

```diff
@@ -345,6 +345,28 @@ load _helpers
   [ "${actual}" = "testing" ]
 }
 
+#--------------------------------------------------------------------
+# podLabels
+
+@test "server/StatefulSet: no pod labels defined by default" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-statefulset.yaml  \
+      . | tee /dev/stderr |
+      yq -r '.spec.template.metadata.labels | del(."app") | del(."chart") | del(."release") | del(."component") | del(."hasDNS")' | tee /dev/stderr)
+  [ "${actual}" = "{}" ]
+}
+
+@test "server/StatefulSet: pod labels can be set" {
```

Comment:
Could you add another testcase here which would verify multiple labels getting set if `podLabels` has multiple values set?

Reply:
Sure thing :)